### PR TITLE
Upgrade django-celery version.

### PR DESCRIPTION
**Pull request by doctoryes**
We've moved to an edX fork of django-celery until the Celery 4 upgrade occurs.

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -2,7 +2,7 @@ Change Log
 ----------
 
 ..
-   All enhancements and patches to cookiecutter-django-app will be documented
+   All enhancements and patches to edx-celeryutils will be documented
```
Comment (efischer19) [inline CHANGELOG.rst]: :rofl:

File: CHANGELOG.rst [hunk 2]
```diff
@@ -18,6 +18,31 @@ Unreleased
...
+
+[0.2.4] - 2017-06-20
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* Add management command to fix djcelery tables.
+
+[0.2.1] - 2017-05-22
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* Add ChordableDjangoBackend and testing.
```
Comment (efischer19) [inline CHANGELOG.rst]: I can't believe that the dev that made these commits forgot to update the changelog. What a scrub!

(But really, thanks for updating this!)

File: requirements/test.txt [hunk 3]
```diff
@@ -4,31 +4,25 @@
...
+pytest==3.2.0             # via pytest-catchlog, pytest-cov, pytest-django
+python-dateutil==2.6.1    # via freezegun
 python-memcached==1.58
 pytz==2017.2              # via celery, django
-six==1.10.0               # via freezegun, packaging, python-dateutil, python-memcached, setuptools
-
-# The following packages are considered to be unsafe in a requirements file:
-# setuptools                # via pytest
+six==1.10.0               # via freezegun, python-dateutil, python-memcached
 mock
```
Comment (jmbowman) [inline requirements/test.txt]: Any idea how this got in without a pinned version?

Comment (doctoryes) [inline requirements/test.txt]: Yes - the test-extra.txt file:
https://github.com/edx/edx-celeryutils/blob/master/requirements/test-extra.txt
which is added in here:
https://github.com/edx/edx-celeryutils/blob/master/Makefile#L47-L51
Thoughts?

**Pull request merged**